src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -337,8 +337,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    help='ISO 8601 timespan duration of the key frame interval duration.')
         c.argument('access_token', arg_group='Input', help='A unique identifier for a stream. This can be specified at creation time but cannot be updated. If omitted, the service will generate a unique value.')
         c.argument('description', help='The live event description.')
-        c.argument('ips', ip_range_type, arg_group='Input')
-        c.argument('preview_ips', ip_range_type, arg_group='Preview')
+        c.argument('ips', ip_range_type, arg_group='Input', help='Space-separated IP addresses for access control. Allowed IP addresses can be specified as either a single IP address (e.g. "10.0.0.1") or as an IP range using an IP address and a CIDR subnet mask (e.g. "10.0.0.1/22"). Use "" to clear existing list. Use "AllowAll" to allow all IP addresses. Allowing all IPs is not recommended for production environments.')
+        c.argument('preview_ips', ip_range_type, arg_group='Preview', help='Space-separated IP addresses for access control. Allowed IP addresses can be specified as either a single IP address (e.g. "10.0.0.1") or as an IP range using an IP address and a CIDR subnet mask (e.g. "10.0.0.1/22"). Use "" to clear existing list. Use "AllowAll" to allow all IP addresses.')
```

Comment:
Add `Allowing all IPs is not recommended for production environments`.
Also, move both params to live-event scope so you don't repeat and remove ip_range_type (it's only nargs='+' what you are inheriting.. you can that prop directly or make another argument type)